docs/software/android/usage.mdx

```diff
@@ -89,8 +89,14 @@ In order to start communicating with your mesh, you must select a region. This s
 
 Edit the "Your name", e.g. to be "Mike Bird". This is the name that other people will see, so make it unique within your group. The initials e.g. "MB" should also be unique and will be used to identify you in the message history and on the device screens. Initials, or "short name", can be customized in the Radio configuration - User settings. The four characters displayed after your initials cannot be changed. These are the last four hex digits of the device MAC address. Devices with unset names will display these four characters as the device short name.
 
+Ignore the Anonynmous Usage check box and the "Report Bug" button at the bottom, they are not enabled and you cannot select them.  They are there for decoration purposes only.
```

Comment:
@andrekir is this accurate?

Reply:
no. these are disabled when there is no Google services available.

Reply:
...then it should say it's disabled, otherwise it's decoration.  I'm glad you know why it's disabled, what about the rest of us using the app?